lib/internal/Magento/Framework/DB/Adapter/Pdo/Mysql.php

```diff
@@ -479,6 +480,7 @@ public function rawQuery($sql)
     /**
      * Run RAW query and Fetch First row
      *
+     * @deprecated
```

Comment:
Could you write some explanation why it's deprecated and what we should use instead?

Reply:
Agree with it, according to our practices we can't just deprecate the method, we always should provide information what to use

Reply:
I added  documentation that  we need use methods from interface, in future we need to do it private